src/espIdf/tracing/system-view/panel.ts

```diff
@@ -94,6 +99,12 @@ export class SystemViewPanel {
       <title>System View Trace</title>
     </head>
     <body>
+      <style>
+      @font-face {
+          font-family: "codicon";
+          src: url('${fontsUri}') format('truetype');
+      }
+      </style>
```

Comment:
How about _(if possible)_ we move these instance in the `espCommons.scss` so that we don't have to type this out manually since most of the webviews are importing that scss...?

Reply:
I tried to put it in espCommons.scss but the issue is that the fonts URI needs to be encode using `vscode.asWebviewUri()` to work properly inside the webview.